src/common/search/FilterPlays.jsx

```diff
@@ -242,6 +252,7 @@ const FilterPlays = ({ reset }) => {
           <FilterPlaysModalBody
             filterQuery={modifiedFilterQuery}
             setFilterQuery={setModifiedFilterQuery}
+            onClearAppliedFilters={resetFilter}
```

Comment:
`resetFilter` will change the actual `filterQuery` state which is going to make graphql call and we donot want that. all we do is reset the `modifiedFilterQuery` state and `noOfAppliedFilter` to be zero. so that we donot want the parent to get refresh on clicking the `clear all` button.